### PR TITLE
Fixed issue #8235 BUG: Testimonials shows zero if no testimonials exist for user

### LINKED ISSUE #8235: [BUG] Testimonials shows zero if no testimonial exist for user
### Description

**Instructions for Reproduction:**
1. Begin by logging in to Linkfree.
2. Access the "Account" section and navigate to the testimonials tab.
3. In case you do not have any testimonials, the following message will be displayed.

This issue can be replicated in both the local and production environments.

**Suggested Fix**
_File name: /pages/account/manage/testimonials.js_ 

**I think istead of checking like this :** 
image

**Use conditional operator like this:** 
image

Hopefully this will solve the issue.

### Screenshots

image

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by tanbirali**
## Fixes Issue 
Closes #8235

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
Screenshot (141)

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8314"></a>

### CODE REVIEW

File: pages/account/manage/testimonials.js [hunk 1]
```diff
@@ -108,78 +108,75 @@ export default function ManageTestimonials({ BASE_URL, testimonials }) {
           </h3>
         </div>
 
-        {!testimonialList?.length && (
+        {!testimonialList?.length ? 
+         <div>
+         <div className="flex gap-4">
```
Comment (eddiejaoude) [inline pages/account/manage/testimonials.js:113]: The indentation doesn't look quite right here? Try using the Prettier plugin for VScode, it should fix on save

File: pages/account/manage/testimonials.js [hunk 2]
```diff
@@ -108,78 +108,75 @@ export default function ManageTestimonials({ BASE_URL, testimonials }) {
           </h3>
         </div>
 
-        {!testimonialList?.length && (
+        {!testimonialList?.length ?
```
Comment (eddiejaoude) [inline pages/account/manage/testimonials.js:111]: I think this condition is the wrong way around? This was originally for the alert and now it is for the list?

Comment (tanbirali) [reply pages/account/manage/testimonials.js]: My bad you are right about the condition being the other way around. I have corrected it please review and inform if things are okay this time. @eddiejaoude

**Comment by tanbirali** [comment]
Please maintainer, review this PR

File: pages/account/manage/testimonials.js [hunk 3]
```diff
@@ -108,77 +108,74 @@ export default function ManageTestimonials({ BASE_URL, testimonials }) {
           </h3>
         </div>
 
-        {!testimonialList?.length && (
```
Comment (sanyamx) [inline pages/account/manage/testimonials.js:111]: change this line of code to (testimonialList.length === 0)

File: pages/account/manage/testimonials.js [hunk 4]
```diff
@@ -108,77 +108,74 @@ export default function ManageTestimonials({ BASE_URL, testimonials }) {
...
-          <Alert type="info" message="No Testimonials found" />
-        )}
-
-        {testimonialList?.length && (
```
Comment (sanyamx) [inline pages/account/manage/testimonials.js:115]: this to (testimonialList.length !== 0)

**Review by sanyamx — COMMENTED** [review]
These two changes in the line should remove the error you are getting from zero. It is how I was able to solve it in a optimal way.

**Comment by sanyamx** [comment]
> Please maintainer, review this PR

I think you can solve this issue with just changes in just 2 lines of code. That are 111 line and 115 line of the page

**Comment by tanbirali** [comment]
I am getting a merge conflict and I need help to resolve

File: pages/account/manage/testimonials.js [hunk 5]
```diff
@@ -49,7 +49,7 @@ export async function getServerSideProps(context) {
 export default function ManageTestimonials({ BASE_URL, testimonials }) {
   const [reorder, setReorder] = useState(false);
   const [showNotification, setShowNotification] = useState(false);
-  const [testimonialList, setTestimonialList] = useState(testimonials || []);
+  const [testimonialList, setTestimonialList] = useState([]);
```
Comment (eddiejaoude) [inline pages/account/manage/testimonials.js:52]:
```suggestion
  const [testimonialList, setTestimonialList] = useState(testimonials || []);
```

**Comment by eddiejaoude** [comment]
Thanks for contributing 👍 

> I am getting a merge conflict and I need help to resolve

I have resolved the conflicts and simplified the changes @tanbirali

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Pull request merged**